lemur/certificates/views.py

```diff
@@ -511,23 +512,21 @@ def post(self, data=None):
         roles.append(role)
         authority_permission = AuthorityPermission(data["authority"].id, roles)
 
-        if authority_permission.can():
-            data["creator"] = g.user
+        if not authority_permission.can():
+            return dict(message=f"You are not authorized to use the authority: {data['authority'].name}"), 403
+
+        data["creator"] = g.user
+        try:
+            service.allowed_issuance_for_domain(data["common_name"], data["extensions"])
```

Comment:
same question as above about the outcome of the authZ eval

Reply:
It raises `UnauthorizedError` if authz fails ([additional details](https://github.com/Netflix/lemur/pull/3889/files#r766258425)), which gets caught below to return 403 along with error message